compiler/test/suites/strings.re

```diff
@@ -107,7 +107,11 @@ describe("strings", ({test, testSkip}) => {
   );
   assertParseWithLocs(
     "string_loc_multi_line",
-    "\"foo\nbar\nbaz\nqux\nquux\"",
+    {|"foo
+bar
+baz
+qux
+quux"|},
```

Comment:
I see, so the indentation here is taken into account with the alternative string syntax. 🤔

Reply:
Hmm, I guess you have too.  It doesn't look right, even though it is right.